README.md

```diff
@@ -12,6 +12,35 @@ More design details can be explored in the [WMCO enhancement](https://github.com
 ## Pre-requisites
 * OKD/OCP 4.6 cluster running on Azure or AWS, configured with [hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md)
 
+### Pre-configuration steps in case of vSphere cluster:
+#### Windows VM assumptions:
+- An updated version of Windows Server 1909 VM image and corresponding container images are used
+- The Windows image is configured with SSH so that WMCO can access it:
+  - SSH on the Windows VM is configured using the [powershell script](https://gist.githubusercontent.com/ravisantoshgudimetla/087bf49088f70edf7f97777386d12f18/raw/0f13048697130a0265ed1050591361fdebcee3ae/powershell.yaml)
+  - A new file called C:\Users\Administrator\.ssh\authorized_keys containing the public key to be used is created
+- VMWare tools is installed, configured and running on the Windows VM:
+  - The [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.1.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) file at C:\ProgramData\VMware\VMware Tools\tools.conf has the following entry to ensure that the cloned vNIC generated on the Windows VM by hybrid-overlay won't be ignored and the VM has an IP Address in the vCenter
+    - `exclude-nics=`
+  - The tools Windows service is running in the Windows VM.
+- The Windows image has been sysprep'ed and it is available for cloning using the following command:
```

Comment:
`The Windows image has been sysperp'ed and made ready for clone using the following command:`